src/drone/scan_trajectory.py

```diff
@@ -11,6 +12,7 @@ def __init__(self, polygon_coords: list, fov_x: float, fov_y: float):
         self.start_point = (0, 0)
         self.fov_x = fov_x
         self.fov_y = fov_y
+        polygon_coords = [(-y, x) for x, y in polygon_coords]
```

Comment:
Mixing coordinates order/values here and in environment is a bit confusing, consider refinement